src/clr-addons/components.clr-addons.scss

```diff
@@ -367,4 +380,25 @@ input:not([type='time']) {
   .content-panel-content-wrapper {
     padding: 0.5rem;
   }
+
+  //prevents the upper body part from being hidden by the fixed header
+  body {
+    padding-top: $clr-header-height;
```

Comment:
padding on the body, will move the whole page downwards with empty space at the top (at least in my applications)